src/main/resources/alma/fix/relatedRessourcesAndLinks.fix

```diff
@@ -611,6 +611,20 @@ end
 
 replace_all("exampleOfWork.label","<<|>>","")
 
+# 041 h|k - Language code of original and/or intermediate translations of text
+if exists("130??|240??")
+  set_array("lang_temp")
+  copy_field("041[ 01] .[hk]", "lang_temp.$append")
+  set_array("exampleOfWork.language[]")
+  do list(path:"lang_temp", "var": "$i")
+    copy_field("$i","exampleOfWork.language[].$append.id")
+    copy_field("$i","exampleOfWork.language[].$last.label")
+  end
+  lookup("exampleOfWork.language[].*.label","ISO639-2-to-GND")
+  prepend("exampleOfWork.language[].*.id", "http://id.loc.gov/vocabulary/iso639-2/")
+end
+
+
 # following only with second indicator 2 for analytical entries:
```

Comment:
I don't understand the "fix"es good enough: is this doubling of code (see [otherFields.fix](https://github.com/hbz/lobid-resources/pull/1968/files#diff-f301681772db7e84b802becdca562362df634a853f1d3943d3bbe9e0382cad2e) ) intended?

Reply:
Maybe @fsteeg can wheigh in how to best implement this?

Reply:
I hove no context here, but if the whole body of the `if` (in relatedRessourcesAndLinks.fix) and `unless` (in otherFields.fix) is identical, maybe it could be extracted to a macro, or just to a separate fix file?

Reply:
The superfluous lines of code have been removed.